tests/vc-generator/TestLinkedDataSignature.js

```diff
@@ -92,15 +88,12 @@ export class LinkedDataSignature extends LinkedDataProof {
   }
 
   /**
-   * @param document.document
-   * @param {object} document - Document to be signed.
-   * @param purpose - {ProofPurpose}.
-   * @param documentLoader - {function}.
-   * @param expansionMap - {function}.
+   * @param {object} options - Options to use.
+   * @param {object} options.document - Document to be signed.
+   * @param {object} options.purpose -  A JSON-LD purpose of the proof.
+   * @param {Function} options.documentLoader - A JSON-LD document loader.
+   * @param {Function} options.expansionMap - A JSON-LD Expansion map.
    *
-   * @param document.purpose
-   * @param document.documentLoader
-   * @param document.expansionMap
    * @returns {Promise<object>} Resolves with the created proof object.
    */
   async createProof({document, purpose, documentLoader, expansionMap}) {
```

Comment:
`expansionMap` is deprecated and should probably just be removed here.

Reply:
thanks for the note updated `@db/vc`, `jsonld`, and `jsigs` and then removed the expansionMap. https://github.com/w3c-ccg/di-ed25519-test-suite/pull/32/commits/202312f0bd378638bc38281b278fbbceb5d74d7a